sampleFiles/sample-schema.json

```diff
@@ -1409,6 +1410,89 @@
               }
             }
           ]
+        },
+        {
+          "name": "hormone_therapy",
+          "description": "The collection of data elements describing the details of a hormone treatment therapy completed by a donor.",
+          "meta": {
+            "parent": "treatment"
+          },
+          "fields": [
+            {
+              "name": "program_id",
+              "valueType": "string",
+              "description": "Unique identifier of the ARGO program.",
+              "meta": {
+                "primaryId": true,
+                "foreignKey": "sample_registration.program_id"
+              },
+              "restrictions": {
+                "required": true,
+                "regex": "[A-Za-z0-9\\-\\._]{1,64}"
+              }
+            },
+            {
+              "name": "submitter_donor_id",
+              "valueType": "string",
+              "description": "Unique identifier of the donor, assigned by the data provider.",
+              "meta": {
+                "primaryId": true,
+                "foreignKey": "sample_registration.submitter_donor_id"
+              },
+              "restrictions": {
+                "required": true,
+                "regex": "[A-Za-z0-9\\-\\._]{1,64}"
+              }
+            },
+            {
+              "name": "submitter_treatment_id",
+              "valueType": "string",
+              "description": "Unique identifier of the treatment, as assigned by the data provider.",
+              "meta": {
+                "primaryId": true,
+                "foreignKey": "treatment.submitter_treatment_id"
+              },
+              "restrictions": {
+                "required": true,
+                "regex": "[A-Za-z0-9\\-\\._]{1,64}"
+              }
+            },
+            {
+              "name": "hormone_therapy_drug_name",
+              "description": "Name of agent or drug administered to patient as part of the hormone therapy treatment regimen.",
+              "valueType": "string",
+              "restrictions": {
+                "required": true,
+                "codeList": ["Placeholder list 1", "Need list", "Still need it", "Other"]
```

Comment:
this codelist was like this from here: https://github.com/icgc-argo/argo-dictionary/blob/master/schemas/hormone_therapy.json. I added "Other"  in hopes that we won't need to change it later :man_shrugging: ...